package.json

```diff
@@ -94,7 +94,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "3.2.4",
     "prettier-plugin-svelte": "3.1.2",
-    "svelte": "^4.2.10",
+    "svelte": "^3 || ^4 || ^5",
```

Comment:
I inadvertently changed this in #321. Keeping the dev version wide makes it easier to switch between versions locally when testing